types/cytoscape-euler/index.d.ts

```diff
@@ -0,0 +1,170 @@
+// Type definitions for cytoscape-euler 1.2
+// Project: https://github.com/cytoscape/cytoscape.js-euler
+// Definitions by: Roger Dubbs <https://github.com/RogerDubbs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import cytoscape = require('cytoscape');
+
+declare const cytoscapeEuler: cytoscape.Ext;
+export = cytoscapeEuler;
```

Comment:
```suggestion
export = cytoscapeEuler;
export as namespace cytoscapeEuler;
```
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
(check the published package)
https://unpkg.com/browse/cytoscape-euler@1.2.2/webpack.config.js

Reply:
Thanks for the suggestion - I incorporated it.